br-ext/package/optee_rust_examples_ext/optee_rust_examples_ext.mk

```diff
@@ -5,11 +5,23 @@ OPTEE_RUST_EXAMPLES_EXT_SITE_METHOD = local
 OPTEE_RUST_EXAMPLES_EXT_INSTALL_STAGING = YES
 OPTEE_RUST_EXAMPLES_EXT_DEPENDENCIES = optee_client_ext
 OPTEE_RUST_EXAMPLES_EXT_SDK = $(BR2_PACKAGE_OPTEE_RUST_EXAMPLES_EXT_SDK)
+OPTEE_RUST_EXAMPLES_EXT_TC_PATH = $(subst $\",,$(BR2_PACKAGE_OPTEE_RUST_EXAMPLES_EXT_TC_PATH))
 
 EXAMPLE = $(wildcard examples/*)
 
+define OPTEE_RUST_EXAMPLES_EXT_CONFIGURE_CMDS
+	@echo Configuring OP-TEE rust examples && \
+	export RUSTUP_HOME=$(OPTEE_RUST_EXAMPLES_EXT_TC_PATH)/.rustup && \
+	export CARGO_HOME=$(OPTEE_RUST_EXAMPLES_EXT_TC_PATH)/.cargo && \
+	source $(OPTEE_RUST_EXAMPLES_EXT_TC_PATH)/.cargo/env && \
+	cd $(@D) && cargo --version >/dev/null
```

Comment:
Is `cargo --version >/dev/null` useful? Just to make sure the env (PATH) is OK?

Reply:
Yeah, it is actually triggering the Rust toolchain installation via: https://github.com/apache/incubator-teaclave-trustzone-sdk/blob/no-std/rust-toolchain.toml

Reply:
See corresponding usage here too: https://github.com/apache/incubator-teaclave-trustzone-sdk/blob/no-std/setup.sh#L35

Reply:
`cargo --version` is triggering some installation?!? That's brain-damaged if you want my opinion, but well... :wink: Fine.

Reply:
We should invent an acronym for that.... PBSE (Programming By Side-Effects)? :wink:

Reply:
Haha, have a read here regarding Rust toolchain overrides: https://rust-lang.github.io/rustup/overrides.html

Reply:
BTW, let me comment it properly too.